autoload/jedi.vim

```diff
@@ -489,6 +489,8 @@ function! jedi#complete_opened(is_popup_on_dot)
         " option is set.
         if g:jedi#popup_select_first && stridx(&completeopt, 'longest') > -1
             return "\<Down>"
+        else
+            return ""
```

Comment:
"popup_on_dot" is only used to trigger the completion (and therefore not relevant here), right?